### ISSUE TITLE
[BUG] Can not create database on phpmyadmin. No privileges to create databases.

### ISSUE THREAD

**Issue by trelljaret** [sprintcube/docker-compose-lamp#136]
This is the error I get when I try to write a `CREATE DATABASE` query.

```
CREATE DATABASE mynewdb
MySQL said: Documentation

#1044 - Access denied for user 'docker'@'%' to database 'mynewdb'
```

And this is the first page of phpmyadmin:

![image](https://user-images.githubusercontent.com/63227135/87239548-a5a35400-c425-11ea-8617-6b017a2227f3.png)

Please help on how to create a new database.

**Comment by CountParadox** [comment]
Have you tried accessing mysql as user root?

when inside the container, run
```
#sudo mysql
mysql>CREATE DATABASE mynewdb
```

**Comment by trelljaret** [comment]
> Have you tried accessing mysql as user root?
> 
> when inside the container, run
> 
> ```
> #sudo mysql
> mysql>CREATE DATABASE mynewdb
> ```

Yes, have done this at that time.
Db is created, But phpmyadmin is not affected (as I remember). It does not show the new database.
The problem is that mysql user `docker` can not craete database. And phpmyadmin is logged in as user `docker`, not `root`.

Anyway, I use mostly this project for testing. But I thought using this in production, so I wanted to know how to add new database.

**Comment by Oddisey000** [comment]
Actually there is a solution.
in **docker-compose.yml** file you need to specify 2 connection variables in **phpmyadmin:** part:
`PMA_USER: root
 PMA_PASSWORD: ${MYSQL_ROOT_PASSWORD}`

**Comment by Taalaybek** [comment]
maybe ist's time to fix this issue?

**Comment by MrOffline77** [comment]
Let me know if this issue is fixed know

**Comment by f-ricci** [comment]
@MrOffline77 this fix is not available on branch 7.4.x

**Comment by MrOffline77** [comment]
@f-ricci thats because the branch based php version model is legacy and not supported anymore. Please use the master branch. More information about this is in the master readme and in the branch readme's.
